### PR TITLE
Update Istio to 1.6.4

### PR DESCRIPTION
**Pull request by mvladev**
**How to categorize this PR?**

/area control-plane
/kind enhancement 
/priority normal

**What this PR does / why we need it**:

Update Istio to 1.6.4. 
Fix CVE-2020-8663 by setting `overload.global_downstream_max_connections`.

Listener and cluster buffer limit to 32KiB.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
Update Istio to `1.6.4`. Fix CVE-2020-8663 by setting `overload.global_downstream_max_connections` on the istio-ingress gateway.
```

### CODE REVIEW

File: charts/istio/istio-ingress/templates/deployment.yaml [hunk 1]
```diff
@@ -127,9 +127,14 @@ spec:
           value: sni-dnat
         - name: ISTIO_META_CLUSTER_ID
           value: "Kubernetes"
+        - name: ISTIO_BOOTSTRAP_OVERRIDE
+          value: /etc/istio/custom-bootstrap/custom_bootstrap.yaml
         volumeMounts:
         - mountPath: /var/run/secrets/istio
           name: istiod-ca-cert
+        - mountPath: /etc/istio/custom-bootstrap
```
Comment (rfranzke) [inline charts/istio/istio-ingress/templates/deployment.yaml:136]: Can you add a checksum annotation for this configmap mount?

Comment (mvladev) [reply charts/istio/istio-ingress/templates/deployment.yaml:136]: fixed in e71bb390e

**Review by timuthy — COMMENTED** [review]
Looks good. Let me run a few tests locally, so we can merge it.

**Review by timuthy — APPROVED** [review]
/lgtm

**Pull request merged**